### PR TITLE
:memo: Add issue handling information

### PR DESCRIPTION
**Pull request by MrKrisKrisu**
What do you say?
@HerrLevin @jeyemwey @xanderio 

belongs to #1421

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # [Träwelling](https://traewelling.de)
 
 > Träwelling is a free check-in service that lets you tell your friends where you are and where you can log your public
-> transit journeys. In short, you can check into trains and get points for it. 
+> transit journeys. In short, you can check into trains and get points for it.
```
Comment (jeyemwey) [inline README.md:4]: I don't understand the diff

**Review by jeyemwey — APPROVED** [review]
Looks good to me. Set expectations which is good.

File: README.md (hunk 1 above)
Comment (MrKrisKrisu) [reply README.md:4]: Removed a space at the end :D

**Review by xanderio — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jeyemwey) [reply README.md:4]: Sorry, wasn't displayed on mobile

**Pull request merged**